### PR TITLE
Support AMP by CA A.J.A. Infeed

### PR DESCRIPTION
**Pull request by i02302**
Hello.
We are A.J.A., a subsidiary of CyberAgent which provides ad-network service in Japan. (https://www.cyberagent.co.jp/)
We'd like to add support to amp-ad.
Can you check if this pull request can be merged?
Thanks.

### CODE REVIEW

**Comment by i02302** [comment]
I signed it! 
Company Name : A.J.A.

**Comment by i02302** [comment]
I signed it!
Corporation Name: CyberAgent

**Comment by i02302** [comment]
CyberAgent

**Comment by i02302** [comment]
I signed it!
Corporation Name: A.J.A.

**Comment by i02302** [comment]
A.J.A.

**Comment by lannka** [comment]
@i02302 I still don't see A.J.A on our CLA list. If it's the same as CyberAgent, can you reply with CyberAgent?

**Comment by i02302** [comment]
@lannka I can do it!

**Comment by i02302** [comment]
CyberAgent

File: ads/caajainfeed.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+import {loadScript, validateData} from '../3p/3p';
+
+/**
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+export function caajainfeed(global, data) {
+
+  validateData(
+  data,
```
Comment (lannka) [inline ads/caajainfeed.js]: please indent 2 more spaces

**Comment by lannka** [comment]
OK, let's forget about the CLA for now...

Could you also implement render-start and no-content API? https://github.com/ampproject/amphtml/blob/master/ads/README.md#developer-guidelines-for-a-pull-request

**Comment by i02302** [comment]
@lannka 
We do not implement render-start and no-content API in this pull request.
Could you please permit it?

File: ads/caajainfeed.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+      'ipv4',
+      'ipv6',
+      'network-reachability',
+      'os-name',
+      'os-version',
+      'os-lang',
+      'os-timezone',
+      'device-version',
+      'app-id',
+      'app-version',
```
Comment (lannka) [inline ads/caajainfeed.js]: all the variable should be camel case: appVersion, appId

Comment (i02302) [inline ads/caajainfeed.js]: Thanks, I'll fix it.

**Comment by erwinmombay** [comment]
@lannka done

**Pull request merged**